src/components/awards/awards.module.scss

```diff
@@ -0,0 +1,78 @@
+.awards {
+  @apply mt-20 flex flex-col;
+
+  &__headers {
+    @apply container flex flex-col justify-center;
+  }
+
+  &__header {
+    @apply text-center text-5xl font-medium leading-tight;
+
+    @screen sm {
+      @apply text-7xl leading-tight tracking-tight;
+    }
+  }
+
+  &__subheader {
+    @apply mx-auto mt-6 w-full px-2 text-center text-lg leading-8;
+
+    @screen sm {
+      @apply text-xl;
+    }
+
+    @screen lg {
+      @apply w-2/3;
+    }
+  }
+
+  &__hero {
+    @apply flex flex-wrap justify-center;
+
+    @screen lg {
+      @apply justify-start;
+    }
+  }
+
+  &__circle {
+    @apply relative left-0 z-10 mt-4 inline-block transform rounded-full border-4 border-solid border-white bg-dark-900;
+  }
+
+  &_honors {
+    @apply flex w-full bg-dark-900 py-6 text-sm font-normal leading-6;
+
+    &__header {
+      @apply rotate-180 transform pl-[52px] font-playfairDisplay text-2xl italic text-white opacity-50 vertical-lr
+      after:absolute after:left-5 after:top-1/2 after:h-[1px] after:w-6 after:bg-white;
+
+      @screen md {
+        @apply pl-[105px];
+        @apply after:left-4 after:w-16;
+      }
+    }
```

Comment:
if you are in the awards section and use BEM, you need to name class awards__honors => awards__honors-header or awards__honors_header etc.